docs/ups_configuration.rst

```diff
@@ -0,0 +1,24 @@
+UPS Carrier Configuration
+=========================
+
+Before any shipment can be sent to the customer address, proper details about
```

Comment:
Before labels can be generated using this module, it has to be configured with your UPS account details.